### PR TITLE
Make new hire access list easier to find

### PR DESCRIPTION
**Pull request by danspndl**
- Added "access to services" section to make the list easier to find
- Added new iOS Slack channel to the list
- Changed Ruby version to current we use
- Replaced Rui in access list

### CODE REVIEW

File: Cookbook/Technical-Documents/ToolsAndServices.md [hunk 1]
```diff
@@ -17,22 +17,21 @@
...
 
   If you want you can install additional git hooks which will check for pods and Xcode templates to be in sync when switching branches or pulling from remote. To install them follow instructions in `githooks/README.md`
-  
+
 * Slack bots
-  
-  You can trigger CI jobs from Slack in the `ios-build` channel using slash commands. Check [this guide](TBD) for details.
-  
+
+  You can trigger CI jobs from Slack in the `ios-build` channel using slash commands. Check [this guide](TBD) for details.
```
Comment (diegopetrucci) [inline Cookbook/Technical-Documents/ToolsAndServices.md]: If there is no guide let's remove this step.

```suggestion
  You can trigger CI jobs from Slack in the `ios-build` channel using slash commands.
```

Comment (diegopetrucci) [reply Cookbook/Technical-Documents/ToolsAndServices.md]: Or maybe @ilyapuchka has written one

Comment (sdaniel55) [reply Cookbook/Technical-Documents/ToolsAndServices.md]: yep [here](https://github.com/Babylonpartners/ios-playbook/blob/master/Cookbook/Technical-Documents/SlackCIIntegration.md). I'll add that

File: Cookbook/Technical-Documents/ToolsAndServices.md [hunk 2]
```diff
@@ -17,22 +17,21 @@
...
 
   If you want you can install additional git hooks which will check for pods and Xcode templates to be in sync when switching branches or pulling from remote. To install them follow instructions in `githooks/README.md`
-  
+
 * Slack bots
-  
-  You can trigger CI jobs from Slack in the `ios-build` channel using slash commands. Check [this guide](TBD) for details.
-  
+
+  You can trigger CI jobs from Slack in the `ios-build` channel using slash commands. Check [this guide](https://github.com/Babylonpartners/ios-playbook/blob/master/Cookbook/Technical-Documents/SlackCIIntegration.md) for details.
```
Comment (sdaniel55) [inline Cookbook/Technical-Documents/ToolsAndServices.md]:
```suggestion
  You can trigger CI jobs from Slack in the `ios-build` channel using slash commands. Check [this guide](./Technical-Documents/SlackCIIntegration.md) for details.  
```

**Review by diegopetrucci — APPROVED** [review]

File: Cookbook/Technical-Documents/ToolsAndServices.md [hunk 3]
```diff
@@ -17,22 +17,21 @@
...
 
   If you want you can install additional git hooks which will check for pods and Xcode templates to be in sync when switching branches or pulling from remote. To install them follow instructions in `githooks/README.md`
-  
+
 * Slack bots
-  
-  You can trigger CI jobs from Slack in the `ios-build` channel using slash commands. Check [this guide](TBD) for details.
-  
+
+  You can trigger CI jobs from Slack in the `ios-build` channel using slash commands. Check [this guide](./Technical-Documents/SlackCIIntegration.md) for details.
```
Comment (sdaniel55) [inline Cookbook/Technical-Documents/ToolsAndServices.md]:
```suggestion
  You can trigger CI jobs from Slack in the `ios-build` channel using slash commands. Check [this guide](SlackCIIntegration.md) for details.  
```

File: Cookbook/Technical-Documents/ToolsAndServices.md [hunk 4]
```diff
@@ -4,11 +4,11 @@
...
-| JIRA | Tickets of tasks and issues on your sprint | You should get an invitation email to join the BabylonPartners team after IT support ticket has been resolved [IT Support ticket](https://supportbybabylon.atlassian.net/servicedesk/customer/portal/5/group/12/create/43) |
+| Slack | Chat with whole org. See channels of interest [here](NewHiresCheckList.md#slack). | You should have given access during your mini induction. Log in with your Babylon email and Okta |
+| 1Password | Access shared passwords used by various services in the team | [IT Support ticket];(https://supportbybabylon.atlassian.net/servicedesk/customer/portal/5/group/12/create/43) Specify in the ticket that access to iOS team vault is needed |
+| JIRA | Tickets of tasks and issues on your sprint | You should get an invitation email to join the BabylonPartners team after the [IT Support ticket](https://supportbybabylon.atlassian.net/servicedesk/customer/portal/5/group/12/create/43) has been resolved |
 | Zeplin | Screen Designs | Ask `@Carlos` or `@adrian` on Slack for access to all the front-end projects |
```
Comment (daniloaliberti) [inline Cookbook/Technical-Documents/ToolsAndServices.md]: `@Carlos` left the company

File: Cookbook/Technical-Documents/ToolsAndServices.md [hunk 5]
```diff
@@ -17,22 +17,21 @@
 | [CircleCI](https://circleci.com/gh/Babylonpartners) | See CI jobs and logs | Use your GitHub login |
 | [Pull Panda](https://pullreminders.com) | Stats on PRs and reminders for when you've been asked for a review | Use your GitHub login |
 | [HockeyApp](https://rink.hockeyapp.net/) | see OTA builds sent internally to QA | Ask support engineer in the iOS team channel on Slack to invite you to the org |
```
Comment (daniloaliberti) [inline Cookbook/Technical-Documents/ToolsAndServices.md]: HockeyApp.net (AppCenter.ms)

**Review by daniloaliberti — APPROVED** [review]
I pointed a few small details. 
Good one! ✨

File: Cookbook/Technical-Documents/ToolsAndServices.md (hunk 4 above)
Comment (sdaniel55) [reply Cookbook/Technical-Documents/ToolsAndServices.md]: What about Adrian?

File: Cookbook/Technical-Documents/ToolsAndServices.md (hunk 5 above)
Comment (sdaniel55) [reply Cookbook/Technical-Documents/ToolsAndServices.md]: Will add this 👀

File: Cookbook/Technical-Documents/ToolsAndServices.md (hunk 4 above)
Comment (sdaniel55) [reply Cookbook/Technical-Documents/ToolsAndServices.md]: Actually, anyone who already has access to the project can invite new members

**Pull request merged**